### PR TITLE
Lucky Number in C++

### PR DESCRIPTION
**Pull request by Anu-123-gif**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Lucky Number is a number that consists of digits 7 and 8 only. 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#2852

### CODE REVIEW

File: C-Plus-Plus/cp/Lucky_Numbers.cpp [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+*/
+    #include <iostream>
+    using namespace std;
+    int main()
+    {
+    int n;//store the number of digits
+    long long a=1, sum=0;
+	
+    cin>>i;
```
Comment (iamrajiv) [inline C-Plus-Plus/cp/Lucky_Numbers.cpp:41]: @Anu-123-gif `i` is undeclared.

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@Anu-123-gif Add space and time complexity. Also, add the problem link. Update the readme in this PR do not make separate PR for the readme file.

File: C-Plus-Plus/cp/Lucky_Numbers.cpp [hunk 2]
```diff
@@ -0,0 +1,57 @@
+/*
+
+Link to problem: https://github.com/TesseractCoding/NeoAlgo/blob/master/C-Plus-Plus/cp/Lucky_Numbers.cpp
```
Comment (iamrajiv) [inline C-Plus-Plus/cp/Lucky_Numbers.cpp]: @Anu-123-gif Add a link to the website or platform where this problem exists.

**Review by iamrajiv — CHANGES_REQUESTED** [review]

**Review by iamrajiv — APPROVED** [review]

**Comment by anshmishra010** [comment]
@ankitaggarwal23 @HarshCasper .
I am refactoring this code .
Please guide me if I did anything wrong as it's my first refactor .

**Pull request merged**